contracts/bondingcurve/FundingPoolMock.sol

```diff
@@ -11,6 +12,6 @@ contract FundingPoolMock {
 
   function allocateFunds(address to, uint256 value) public {
     artistToken.fundsAllocated(value);
-    artistToken.transfer(to, value);
+    IERC20(artistToken.externalToken()).transfer(to, value);
```

Comment:
This seems to solve the issue but we need to polish this function because this opens the FundingPool to a security vulnerability as an attacker can withdraw all funding pool funds to his own malicious account. I am creating a separate test for this and will open a new PR.